packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/controller.js

```diff
@@ -59,4 +59,8 @@ export default class KubernetesNodesCtrl {
         );
       });
   }
+
+  static displayUpdateToMonthlyAction(node) {
+    return node.billingType === 'hourly' && node?.planCodes?.monthly;
```

Comment:
```suggestion
    return node?.billingType === 'hourly' && node?.planCodes?.monthly;
```

Reply:
the billingType attribute do always exist since it's set in the same controller.